docs/index.md

```diff
@@ -0,0 +1,48 @@
+---
+template: 'home.html'
+title: Djinni
+hide:
+  - navigation
+  - toc
+---
+
+
+## :clap: Simple IDL
+
+The [**Djinni-IDL**](djinni-generator/idl.md) provides a simple, yet powerful way to design your interfaces.
+
+---
+
+## :electric_plug: IDE Integration
+
+Plugins for the Djinni IDL are available for [**IntelliJ IDEA**](djinni-intellij-plugin/about.md) & [**VS Code**](vscode-plugin.md)
+
+---
+
+## :rocket: Ready for the Future
+
+Djinni is also compatible with modern technologies. Call your C++ Code from **Kotlin** and **Swift** to have even more fun!
+
+
+---
+
+## :arrow_up_down: Works in both directions
+
+Call in and out of C++ with ease:
+
+```plantuml
+@startuml
```

Comment:
plantuml ftw! It renders on the server side, thats why I chose plantuml over mermaid. It seems to be pretty powerful 😮